### DISCUSSION TITLE
Changing default theme to Dark

### DISCUSSION THREAD

**Discussion by PopulusDev** [hbstack/theme#755] [Q&A] [answered]
Hi

I was wondering if there's an easy way to change the default theme to Dark, default is White originally I believe.

**Comment by razonyang** [comment, marked as answer]
You can  enable dark mode only, see https://hbstack.dev/docs/look-and-feel/enable-dark-mode-only/.

Currently it's not available to enable themes toggle with dark mode as default.

**Reply by PopulusDev** [reply]
Ok, thanks
